common/reconciliation/invariant/timerInvalid.go

```diff
@@ -61,7 +61,6 @@ func (h *TimerInvalid) Check(
 			Info:            "failed to check: expected timer entity",
 		}
 	}
-
```

Comment:
I think you'll need to do the same domainCache change here as in historyExists.go and openCurrenntExecution.go.
Are you planning to do in another PR?